github/apps_installation.go

```diff
@@ -10,14 +10,28 @@ import (
 	"fmt"
 )
 
+// InstallationPermissions Lists the permissions for metadata, contents, issues and single file for an installation.
```

Comment:
I'm noticing that the only file in our repo that uses type `Installation` is [apps.go](https://github.com/google/go-github/blob/master/github/apps.go#L71-L123), so I'm thinking that we should move `InstallationPermissions` and `Installation` into apps.go.

Also, for Godoc purposes, please change `Lists` to `lists` here so that this reads as a normal English sentence.

Reply:
>I'm noticing that the only file in our repo that uses type Installation is apps.go, so I'm thinking that we should move InstallationPermissions and Installation into apps.go.

@gmlewis no strong opinions here. I can move it over to `apps.go` if that makes more sense.

Reply:
>Also, for Godoc purposes, please change Lists to lists here so that this reads as a normal English sentence.

👍

Reply:
Yes, please move those two structs to `apps.go` in this PR if you don't mind.
Thank you!

Reply:
done in https://github.com/google/go-github/pull/861/commits/ba66189c58f247ec0277709cbb2f3b90f9c4dd82